tooling/measures/nx-performance/audit/project-graph.audit.ts

```diff
@@ -39,5 +44,5 @@ export function scoreProjectGraphDuration(duration: number, maxDuration: number)
 export async function projectGraphTiming(): Promise<{ duration: number }> {
   const start = performance.now();
   await execFile('npx nx show projects');
```

Comment:
```suggestion
  execFile('npx nx show projects', { env: {...process.env, NX_DAEMON: false, NX_CACHE_PROJECT_GRAPH: false, NX_ISOLATE_PLUGINS: true });
```

This will provide a more absolute value.

Reply:
I see you already used those variables in project-task.audit.ts